src/Application/Actions/Donations/Confirm.php

```diff
@@ -44,6 +45,7 @@ public function __construct(
         private DonationRepository $donationRepository,
         private Stripe $stripe,
         private EntityManagerInterface $entityManager,
+        private RoutableMessageBus $bus,
```

Comment:
this only needs a `\Symfony\Component\Messenger\MessageBusInterface` doesn't it? Confirm doesn't care that it's routable.

Reply:
Done for this class in c0d6d320d245fad4e77560e4cc4ff68e6d4fc19a